src/cephadm/cephadm

```diff
@@ -2297,6 +2312,48 @@ def command_adopt():
             call_throws(['umount', data_dir_src])
         os.rmdir(data_dir_src)
 
+        logger.info('Chowning content...')
+        call_throws(['chown', '-c', '-R', '%d.%d' % (uid, gid), data_dir_dst])
+
+        if daemon_type == 'mon':
+            # rename *.ldb -> *.sst, in case they are coming from ubuntu
+            store = os.path.join(data_dir_dst, 'store.db')
+            num_renamed = 0
+            if os.path.exists(store):
+                for oldf in os.listdir(store):
+                    if oldf.endswith('.ldb'):
+                        newf = oldf.replace('.ldb', '.sst')
+                        oldp = os.path.join(store, oldf)
+                        newp = os.path.join(store, newf)
+                        logger.debug('Renaming %s -> %s' % (oldp, newp))
+                        os.rename(oldp, newp)
+            if num_renamed:
+                logger.info('Renamed %d leveldb *.ldb files to *.sst',
+                            num_renamed)
+        if daemon_type == 'osd':
+            for n in ['block', 'block.db', 'block.wal']:
+                p = os.path.join(data_dir_dst, n)
+                if os.path.exists(p):
+                    logger.info('Chowning %s...' % p)
+                    os.chown(p, uid, gid)
+            # disable the ceph-volume 'simple' mode files on the host
+            simple_fn = os.path.join('/etc/ceph/osd',
+                                     '%s-%s.json' % (daemon_id, osd_fsid))
+            if os.path.exists(simple_fn):
+                new_fn = simple_fn + '.adopted-by-cephadm'
+                logger.info('Renaming %s -> %s', simple_fn, new_fn)
+                os.rename(simple_fn, new_fn)
+                logger.info('Disabling host unit ceph-volume@ simple unit...')
+                call_throws(['systemctl', 'disable',
+                             'ceph-volume@simple-%s-%s.service' % (
+                                 daemon_id, osd_fsid)])
+            else:
+                # assume this is an 'lvm' c-v for now, but don't error
+                # out if it's not.
+                logger.info('Disabling host unit ceph-volume@ lvm unit...')
+                call(['systemctl', 'disable',
```

Comment:
Do these need an additional stop or that is handled above when the OSD is stopped?